content/metadata.yaml

```
@@ -261,6 +287,14 @@ authors:
         contributions:
           - Writing - Review & Editing
           - Visualization
+      vaccines-trad:
+        order: 3
+        contributions:
+          - Project Administration
```

```suggestion
          - Project Administration
          - Writing - Review & Editing          
```
Thank you! I missed this one!